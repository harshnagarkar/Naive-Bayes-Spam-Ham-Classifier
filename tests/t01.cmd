  training_ham.txt training_spam.txt testing_ham.txt testing_spam.txt 1 
