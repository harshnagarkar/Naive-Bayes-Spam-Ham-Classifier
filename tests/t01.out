ham -186.007 -219.517
ham -126.255 -142.947
ham -281.242 -301.387
ham -500.955 -545.246
ham -358.876 -390.278
ham -274.673 -307.886
ham -269.926 -289.844
ham -201.328 -213.549
ham -359.21 -409.606
ham -230.331 -257.675
ham -358.151 -397.82
ham -369.646 -412.305
ham -233.958 -251.65
ham -489.083 -535.594
ham -0.241914 -1.53769
ham -585.429 -591.384
ham -1143.17 -1154.81
ham -739.065 -762.712
ham -504.712 -517.13
spam -149.792 -146.506
spam -108.636 -107.676
ham -221.689 -225.496
ham -391.922 -401.107
spam -94.1671 -92.1079
ham -21.8596 -23.6131
ham -805.333 -821.779
ham -590.725 -609.549
ham -643.487 -660.724
ham -513.057 -525.474
ham -655.955 -674.365
ham -0.241914 -1.53769
1 0.535714 0.580645
